test/Storage/CqlTests.cs

```diff
@@ -0,0 +1,47 @@
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Extensions.Logging;
+using Moq;
+using Snd.Sdk.Storage.Cql;
+using Xunit;
+
+namespace Snd.Sdk.Tests.Storage
+{
+    public class CqlTests : IClassFixture<AkkaFixture>, IClassFixture<LoggerFixture>
+    {
+        private readonly AkkaFixture akkaFixture;
+        private readonly LoggerFixture loggerFixture;
+
+        public CqlTests(AkkaFixture akkaFixture, LoggerFixture loggerFixture)
+        {
+            this.akkaFixture = akkaFixture;
+            this.loggerFixture = loggerFixture;
+        }
+
+        [Theory]
+        [InlineData("1000 per second", true)]
```

Comment:
I would suggest to make a dedicated test for rate limit parsing (with negative cases).